coderd/appearance/appearance.go

```diff
@@ -33,13 +28,33 @@ var DefaultSupportLinks = []codersdk.LinkConfig{
 	},
 }
 
-type AGPLFetcher struct{}
+func DefaultSupportLinks(docsURL string) []codersdk.LinkConfig {
+	if docsURL == "" {
+		docsURL = "https://coder.com/docs/coder-oss"
```

Comment:
Just /docs now

Reply:
should we also include `https://coder.com/docs/@{VERSION}`

Reply:
> should we also include `https://coder.com/docs/@{VERSION}`

Assuming we always have access to that information yeah I think we should.

Reply:
I assume it can be factored in from buildinfo.

Reply:
For context here is the related issue for more info: https://github.com/coder/coder/issues/13369

Reply:
It could also be done in a separate PR to keep the scopes cleaner.

Reply:
No it's okay I think it makes sense to add the right link right now. I updated the PR to used the embedded build info on the frontend to include the version.

Reply:
Can also add fixes #13369 in PR body.

Reply:
This PR doesn't change docs links throughout the site, just the one that gets used in the default support links in the user drop down.
But I can put a PR that does fix #13369 this afternoon/tomorrow morning haha.

Reply:
@matifali https://github.com/coder/coder/pull/14205